du.md

```diff
@@ -0,0 +1,61 @@
+---
+---
+
+du
+-------
+
+The du (disk usage) command shows you the sizes of directory trees including all of their contents and sizes of individual files.
+This makes it usefull for taking down particular directories that are hogging disk space. 
+
+~~~ bash
+$ du
+92	.
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### 'du -h'
+~~~ bash
+$ du -h
```

Comment:
Love it. Nice use of the yes command :)

> On Mar 25 2016, at 11:26 am, Matt Terwilliger
&lt;notifications@github.com&gt; wrote:  

>

> In
[du.md](https://github.com/c4cs/c4cs.github.io/pull/164#discussion_r57454269):

>  
>  
>     > +

>     > +The du (disk usage) command shows you the sizes of directory trees
including all of their contents and sizes of individual files.

>     > +This makes it usefull for taking down particular directories that are
hogging disk space.

>     > +

>     > +~~~ bash

>     > +$ du

>     > +92     .

>     > +~~~

>     > +

>     > +<!--more-->

>     > +

>     > +### Useful Options / Examples

>     > +

>     > +#### 'du -h'

>     > +~~~ bash

>     > +$ du -h

>  
>

> I think this is a particularly useless example. What about something like
this?

>  
>  
>     $ mkdir test

>     $ yes > test/bigfile.txt

>     $ du -h

>     51M    ./test

>     51M    .

>  
>

> —  
You are receiving this because you are subscribed to this thread.  
Reply to this email directly or [view it on GitHub](https://github.com/c4cs/c4
cs.github.io/pull/164/files/414ad33efb2785b9b1f72c85040a9c334e44b15e#r57454269
)![](https://github.com/notifications/beacon/AAUt3pwAYemgmMMIHq8A33o9D9czZX70k
s5pw_60gaJpZM4H4XSD.gif)